.gitignore

```diff
@@ -1,57 +0,0 @@
-# Prerequisites
```

Comment:
I thought these could to be deleted because its unchanged / has no relation with the current branch.  Should I add them back?

Reply:
Yes, you should because once this pull request is merged these files will be deleted from master branch. We don't want that since further commits will contain any of these files, all these files will be deleted permanently. Let's have a call and I will tell you.